style/template.tex

```diff
@@ -152,13 +158,36 @@
 \fi
 $endif$
 
+% $if(csl-refs)$
```

Comment:
What is the purpose of this block?

Reply:
Check out `pandoc -D latex` to see what the standard template currently is.

Reply:
Huh, interesting. Whatever works, and it certainly doesn't seem to be breaking anything!